task/04-arrays-tasks.js

```diff
@@ -116,7 +121,11 @@ function removeFalsyValues(arr) {
  *    [ 'a', 'b', 'c', 'd', 'e', 'f', 'g' ]  => [ 'A', 'B', 'C', 'D', 'E', 'F', 'G' ]
  */
 function getUpperCaseStrings(arr) {
-   throw new Error('Not implemented');
+   let new_arr = arr.map((currentValue, index, array) => {
+      currentValue=array[index].toString().toUpperCase();
+      return currentValue;
+   });
+   return new_arr;
 }
```

Comment:
extra steps in the code
Please, see the first example here https://www.w3schools.com/jsref/jsref_map.asp